libnavigation-base/src/main/java/com/mapbox/navigation/base/logger/LoggerProvider.kt

```diff
@@ -0,0 +1,18 @@
+package com.mapbox.navigation.base.logger
```

Comment:
```suggestion
package com.mapbox.navigation.base.internal.logger
```

or even better, could you move this to the `libnavigation-util` module and also `internal` package?